modules/statics/src/coins.ts

```diff
@@ -740,7 +740,7 @@ export const coins = CoinMap.fromCoins([
   account(
     '6e35c0dc-7ee8-4273-985d-254aa3641c9d',
     'tia',
-    'celestia',
+    'Celestia',
```

Comment:
This would be a breaking change that would need to be separated into its' own commit.